cpp/oneapi/dal/algo/jaccard/backend/cpu/vertex_similarity_default_kernel_avx2.hpp

```diff
@@ -330,9 +330,10 @@ vertex_similarity_result call_jaccard_default_kernel_avx2(
             }
         }
     }
-    vertex_similarity_result res(homogen_table::wrap(first_vertices, 2, number_elements_in_block),
-                                 homogen_table::wrap(jaccard, 1, number_elements_in_block),
-                                 nnz);
+    vertex_similarity_result res(
+        homogen_table::wrap(first_vertices, number_elements_in_block, 2, data_layout::column_major),
+        homogen_table::wrap(jaccard, number_elements_in_block, 1, data_layout::column_major),
+        nnz);
```

Comment:
It's confusing for me - why it works. `data_layout::column_major` does not transpose the data or do another complex logic - it just says that the data stored column by column. Why you swapped the number of rows and columns?

Reply:
It is the requirement to store the first vertices and the second vertices in a continuous arrays. But they have to be represented as [n x 2] table. Column-major format is a good option for this case.